pi-manage

```diff
@@ -106,7 +106,7 @@ manager = Manager(app)
 admin_manager = Manager(usage='Create new administrators or modify existing '
                               'ones.')
 backup_manager = Manager(usage='Create database backup and restore')
-realm_manager = Manager(usage='Create new realms')
+realm_manager = Manager(usage='Create new realms, set the default realm')
```

Comment:
In other usage descriptions we use 'and' or 'or'. I would like better to 'Create new realms or set the default realm.'

Reply:
done. I also added the realm deletion function. I also added simple multiple resolver support for the create command.